### PR TITLE
Mas810 alpha shajsyed feb

### PR DESCRIPTION
**Pull request by shajeena**

### CODE REVIEW

File: azure/bootnode-image/bootnode-image-step-1.sh [hunk 1]
```diff
@@ -71,7 +71,7 @@ echo $output
 vmip=$(echo $output | jq '.publicIpAddress' | tr -d '"')
 echo "VM IP address: $vmip"
```
Comment (aadawadk) [inline azure/bootnode-image/bootnode-image-step-1.sh:73]: the main branch is replaced with mas810-alpha-shajsyed
undo this change

File: azure/master-arm/byol-upi/mainTemplate-dev.json [hunk 2]
```diff
@@ -211,19 +211,32 @@
...
-      ]
+      ],
+      "type": "String",
+      "metadata": {
+        "description": "Select true to create a private OpenShift cluster."
+      }
     },
     "openShiftClustervnetId": {
-      "type": "string",
+      "defaultValue": "kxr601-vnet",
```
Comment (aadawadk) [inline azure/master-arm/byol-upi/mainTemplate-dev.json:228]: vnet name is supposed to be hardcoded?
kxr601-vnet

File: azure/bootnode-image/bootnode-image-step-1.sh (hunk 1 above)
Comment (shajeena) [reply azure/bootnode-image/bootnode-image-step-1.sh:74]: Thanks Amrin , will remove this after my testing is done and my branch is ready for merge

File: azure/master-arm/byol-upi/mainTemplate-dev.json (hunk 2 above)
Comment (shajeena) [reply azure/master-arm/byol-upi/mainTemplate-dev.json:228]: Will remove this all these harcoding , Thanks
